bin/audit-svg-colors.js

```diff
@@ -16,36 +16,69 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
+ *
+ * @returns Sorted array.
+ */
+const sortBy = () => {
+	return ( objA, objB ) => {
+		if ( objA.to.name > objB.to.name ) {
+			return 1;
+		} else if ( objB.to.name > objA.to.name ) {
+			return -1;
+		}
+		return 0;
+	};
+};
 
 /**
  * Palette color subsets
  */
 
+/**
+ * pickBy function is a replacement for the Lodash ._pickby
+ *
+ * @param {*} obj a palette of colors
```

Comment:
If the object is always a palette of colors, should we rename the argument to `palette`?

Reply:
Yes, it will clarify things since we are not using pickBy anywhere else.